src/components/Rankings.js

```diff
@@ -0,0 +1,93 @@
+import React, {Component} from "react";
+import {Header} from "semantic-ui-react";
+import {Helmet} from "react-helmet";
+import * as riderStageConnectionsActions from "../actions/riderStageConnectionsActions";
+import store from "../store";
+import {connect} from "react-redux";
+import OfficalRanking from "./OfficialRanking";
+import VirtualRanking from "./VirtualRanking";
+import PointRanking from "./PointRanking";
+import MountainRanking from "./MountainRanking";
+import {Route, Switch, Link} from 'react-router-dom';
+import {Menu} from 'semantic-ui-react';
+import * as dateUtil from "../util/date.js";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import of 'dateUtil'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS1128)